content/en/apps/reference/contact-page.md

```diff
@@ -20,7 +20,7 @@ Helper variables and functions for the contact summary can be defined in `contac
 | Variable | Description |
 |---|---|
 | `contact` | The currently selected contact. This has minimal stubs for the `contact.parent`, so if you want to refer to a property on the parent use `lineage` below.| 
-| `reports` | An array of reports for the contact or for any of the contact's `person` children. Note that if the contact has more than 500 reports, only the 500 with the latest `reported_date` values will be provided. | 
+| `reports` | An array of reports for the contact or for any of the contact's `person` children. Note that if the contact has more than 500 reports, only the 500 with the latest `reported_date` values will be provided. Added in `4.7.0`. |
```

Comment:
If someone only saw this they'd probably think that this entire `reports` variable was added in 4.7.0. I'd suggest we rephrase this to say something like "Prior to 4.7.0, only 50 reports were provided". We use similar language in the "Note" [here](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/hierarchy/)